themes/accessway/templates/Security.ss

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="$Lang">
+    <head>
+        <% if $SiteConfig.Title %>
+            <title>$SiteConfig.Title: <%t Accessway.LOGIN "Log in" %></title>
+            $Metatags(false).RAW
+        <% else %>
+            $Metatags.RAW
+        <% end_if %>
+        <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+        <% require css("silverstripe/login-forms: client/dist/styles/bundle.css") %>
+    </head>
+    <body>
+        <section>
+            <header>
+                <h1>
+                    $SiteConfig.Title
+                    <% if not $SiteConfig.Title %>SilverStripe<% end_if %>
+                </h1>
+            </header>
+            $Form
+        </section>
+        <footer>
+            <img
+                src="$resourceURL('silverstripe/login-forms:client/img/SilverStripeLogo200.png')"
+                alt="<%t Accessway.SILVERSTRIPELOGO "SilverStripe Logo" %>"
```

Comment:
Please use a namespaced translation key, and perhaps swap Accessway for something like "SilverStripe\LoginForms.SILVERSTRIPE_LOGO" etc